assets/bundles/bundle.properties

```diff
@@ -20,85 +20,88 @@ block.heavymachineryjava-lonsdaleite-compressor.description = Heats up and coppr
 
 //(Effect)
 block.heavymachineryjava-statusEffectProjector.name = Status Effect Projector
-block.heavymachineryjava-statusEffectProjector.description = Heals ally units and burns enemies
+block.heavymachineryjava-statusEffectProjector.description = Heals ally units and burns enemies.
 block.heavymachineryjava-statusEffectProjector.details = Enemies like them… S H O U L D  B E [scarlet]B U R N I N G  I N  H E L L
 
 block.heavymachineryjava-tesla.name = Tesla
-block.heavymachineryjava-tesla.description = Electrocutes units
+block.heavymachineryjava-tesla.description = Electrocutes units.
+#does it inflict shocked buff?
 
 //(Storage)
 block.heavymachineryjava-miniCore.name = Mini Core
-block.heavymachineryjava-miniCore.description = Mini Core can be built anywhere but it cant be destroyed
-,
+block.heavymachineryjava-miniCore.description = The Mini Core can be built anywhere but it can't be destroyed.
+block.heavymachineryjava-miniCore.details = covalence bad, use mini core instead
```

Comment:
hey wai-

Reply:
i forgor :skull: that covalence is for v7 only

Reply:
What